app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextConverter.java

```diff
@@ -101,6 +102,10 @@
 
     public static final String HTML_MERMAID_INCLUDE = "<script src='file:///android_asset/mermaid/mermaid.min.js'></script>";
 
+    public static final String HTML_ADMONITION_INCLUDE = "<link rel='stylesheet'  type='text/css' href='file:///android_asset/flexmark/admonition.css'>" +
+            "<script src='file:///android_asset/flexmark/admonition.js'></script>";
+    public static final String CSS_ADMONITION = CSS_S + ".adm-block { width: initial; font-size: 90%; text-indent: 0em; } .adm-heading { height: auto; padding-top: 0.4em; padding-left: 2.2em; padding-bottom: 0.4em; } .adm-body { padding-top: 0.25em; padding-bottom: 0.25em; margin-left: 0.5em; margin-right: 0.5em; } .adm-heading > span { color: initial; } .adm-icon { position: absolute; top: 50%; left: 0.5em; transform: translateY(-50%); } .adm-block > .adm-heading { position: relative; cursor: pointer; } .adm-block.adm-collapsed > .adm-heading { margin-bottom: 0; } .adm-block.adm-collapsed .adm-body { display: none !important; } .adm-block.adm-open > .adm-heading:after, .adm-block.adm-collapsed > .adm-heading:after { top: 50%; transform: translateY(-50%); content: '▼'; } .adm-block.adm-collapsed > .adm-heading:after { content: '◀'; } pre + div.adm-block, div.adm-block + pre { margin-top: 1.75em; }" + CSS_E;
```

Comment:
if there something where you say, OK this is really special and maybe not so important... it can be also slimmed down. Lesser custom rules the better :D

Reply:
I had three problems with the default CSS:

* Depending on font and font size, content in `div.adm-heading` is not always vertically centered.
* The collapse indicators `▼` and `◀` were not aligned and jumped around, which is irritating and doesn't look nice.
* There was no vertical space between an admonition box and an immediately preceeding or following code block.

The changes in `CSS_ADMONITION` address these issues. I can understand that you prefer less customization, but in this case I think it's neccessary.

Reply:
I'm not about generally removing it. I just want to know if theres something in the the lots custom rules which _is not_ necessary.

Reply:
I cleaned up the CSS a bit and removed unneccessary rules. I would really like to keep the font size and paddings, because it uses space mor efficiently on small displays.

Reply:
It is fine to keep what is neccassary, I really just talked about the unneccassary. Which was good to ask - otherwise nothing would be removed :D